src/modules/main/components/EncourageSection/EncourageSection.jsx

```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types";
+import styles from "./EncourageSection.module.scss";
```

Comment:
I would recommend changing the CSS file name to `section.module.scss` and this file to `Section`.

Reply:
I agree with You that `Section.jsx `and `section.module.scss` should be enough